test/sql-tap/gh-2931-savepoints.test.lua

```diff
@@ -81,13 +81,15 @@ local testcases = {
 		{0,{1,2,10,11,1,2,4,10,11}}},
 	{"14",
 		[[insert into t1 values(4);]],
-		{1,"Duplicate key exists in unique index \"pk_unnamed_T2_1\" in space \"T2\" with old tuple - [4] and new tuple - [4]"}},
+        {1,"Duplicate key exists in unique index \"pk_unnamed_t2_1\" in "..
+           "space \"t2\" with old tuple - [4] and new tuple - [4]"}},
```

Comment:
Typo: I suggest to leave the original indent level.

Reply:
See above.